src/DateTime.cc

```diff
@@ -436,6 +436,9 @@ DateTime::DateTime(std::string const& iso8601, Timescale scale) {
 }
 
 double DateTime::get(DateSystem system, Timescale scale) const {
+    if (!isValid()) {
+        throw LSST_EXCEPT(pexEx::RuntimeError, "DateTime not valid");
```

Comment:
This seems to be a common possibility.  What about adding a custom exception for this situation?